### ISSUE TITLE
Our newsletter sign-up form (in the menu) is difficult to access and navigate using just a keyboard

### ISSUE THREAD

**Issue by sssoz** [oaworks/oa-works#127]
This fails [WCAG 2.1 success criterion 2.1.1: **keyboard**](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html).

**Comment by thinkbulecount2** [comment]
Closing this because it is still navigable via keyboard, it's just that it has to go through the menu first.
